### PR TITLE
Add a blog post about the move to CocoaDocs Colo + Jazzy

### PR DESCRIPTION
**Pull request by orta**
We've never publicly mentioned the Jazzy support, so this should be made public in a more concrete form.

### CODE REVIEW

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- more -->
+
+## History.
+
+CocoaDocs is a server that generates CocoaPods documentation. It started out by generating Apple-compliant DocSets via [appledoc][2] on a hosted Mac and then sends the results to Amazon's S3 where they are served statically.
+
+## Maturation Process
+
+Appledoc is great, but it's not going to be supporting Swift anytime soon. It's a pre-ARC project that isn't compilable with modern Xcodes. CocoaPods has been supporting Swift Pods since xmas 2014, we wanted to be able to provide hosted documentation for these pods too. Sam Giddins took the initiative here and added [early support][3] for Swift Pods via [Jazzy][4]. You can see a few here: [Observable](http://cocoadocs.org/docsets/Observable-Swift/0.4.2/) , [Moya](http://cocoadocs.org/docsets/Moya/0.6.1/), [JSONJoy](http://cocoadocs.org/docsets/JSONJoy-Swift/0.9.1/) and [QRReader.swift](http://cocoadocs.org/docsets/QRCodeReader.swift/3.1.7/). There's no fancy design work yet, we will get to that in time.
```
Comment (kylef) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: Pods*

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "CocoaDocs Co-located"
+author: orta
+categories: cocoadocs.org releases website
+---
+
+TL;DR: [CocoaDocs.org][1] just got Jazzy support, became more stable, and is now sending stats to our status page.
```
Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: `[Jazzy][4]`

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+<!-- more -->
+
+## History.
+
+CocoaDocs is a server that generates CocoaPods documentation. It started out by generating Apple-compliant DocSets via [appledoc][2] on a hosted Mac and then sends the results to Amazon's S3 where they are served statically.
+
+## Maturation Process
+
+Appledoc is great, but it's not going to be supporting Swift anytime soon. It's a pre-ARC project that isn't compilable with modern Xcodes. CocoaPods has been supporting Swift Pods since xmas 2014, we wanted to be able to provide hosted documentation for these pods too. Sam Giddins took the initiative here and added [early support][3] for Swift Pods via [Jazzy][4]. You can see a few Pods here: [Observable](http://cocoadocs.org/docsets/Observable-Swift/0.4.2/) , [Moya](http://cocoadocs.org/docsets/Moya/0.6.1/), [JSONJoy](http://cocoadocs.org/docsets/JSONJoy-Swift/0.9.1/) and [QRReader.swift](http://cocoadocs.org/docsets/QRCodeReader.swift/3.1.7/). There's no fancy design work yet, we will get to that in time.
```
Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: let's not mention christmas, perhaps say `New Year's` instead?

Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: `Samuel Giddins` (also maybe we can sneak a link into Realm here?)

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Given Swift's immaturity, this is going to be an interesting problem, as the CocoaDocs server has no way of determining which version of Swift your library supports. For now this is solved by only supporting stable Xcode releases. In the future, it may make sense to add a range of supported Swift releases to your podspec per-release.
+
+Towards the end of the 2014 the CocoaDocs server was starting to have some real issues, the generator would be down for days, and trying to `ssh` in would fail. It was turning out that the shared VPS we were using was falling over after a certain amount of time, likely due to running RAM. In order to get the server back up and running I had to submit a support ticket and get someone else to reboot it.
```
Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: `2014, the`

Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: `likely due to running out of RAM`

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Around this time frame [Button][5] reached out to me to ask if this was a problem they could help solve. Given that until this time I had been paying for CocoaDocs myself, I had done everything I could to optimise it for my wallet. ( CocoaDocs was costing me about $450 a year. ) So we jumped at the chance to do it right. CocoaDocs is now running on [macminicolo.net][6], and it's a dream to ssh or Screen Share.app in. Thanks a lot [Button][5]!
```
Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: `Around this time, Button`

Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: `ssh`, `Screen Share.app`

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+In the [process](https://github.com/CocoaPods/cocoadocs.org/issues/320) of moving I added one more feature, we now send data to the [CocoaPods status][7] page saying how many pods have been doc'd in the last 24hrs.
```
Comment (segiddins) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: `s/doc'd/documented`

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+<!-- more -->
+
+## History.
+
+CocoaDocs is a server that generates CocoaPods documentation. It started out by generating Apple-compliant DocSets via [appledoc][2] on a hosted Mac and then sends the results to Amazon's S3 where they are served statically.
+
+## Maturation Process
+
+Appledoc is great, but it's not going to be supporting Swift anytime soon. It's a pre-ARC project that isn't compilable with modern Xcodes. CocoaPods has been supporting Swift Pods since New Year 2014, we wanted to be able to provide hosted documentation for these pods too. Sam Giddins took the initiative here and added [early support][3] for Swift Pods via [Realm][8]'s [Jazzy][4]. You can see a few Pods here: [Observable](http://cocoadocs.org/docsets/Observable-Swift/0.4.2/) , [Moya](http://cocoadocs.org/docsets/Moya/0.6.1/), [JSONJoy](http://cocoadocs.org/docsets/JSONJoy-Swift/0.9.1/) and [QRReader.swift](http://cocoadocs.org/docsets/QRCodeReader.swift/3.1.7/). There's no fancy design work yet, we will get to that in time.
```
Comment (jpsim) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: > It's a pre-ARC project that isn't compilable with modern Xcodes

That was true for a long time, but fixed as of a few months ago. Appledoc builds fine with Xcode 5+ now.

Comment (jpsim) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: Extra space before comma in this line:

> You can see a few Pods here: [Observable](http://cocoadocs.org/docsets/Observable-Swift/0.4.2/),

File: _posts/2015-03-23-CocoaDocs-Colo.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+Towards the end of the 2014, the CocoaDocs server was starting to have some real issues, the generator would be down for days, and trying to `ssh` in would fail. It was turning out that the shared VPS we were using was falling over after a certain amount of time, likely due to running out of RAM. In order to get the server back up and running I had to submit a support ticket and get someone else to reboot it.
+
+Around this time, [Button][5] reached out to me to ask if this was a problem they could help solve. Given that until this time I had been paying for CocoaDocs myself, I had done everything I could to optimise it for my wallet. ( CocoaDocs was costing me about $450 a year. ) So we jumped at the chance to do it right. CocoaDocs is now running on [macminicolo.net][6], and it's a dream to ssh or Screen Share.app in. Thanks a lot [Button][5]!
+
+In the [process](https://github.com/CocoaPods/cocoadocs.org/issues/320) of moving I added one more feature, we now send data to the [CocoaPods status][7] page saying how many pods have been documented in the last 24hrs.
```
Comment (jpsim) [inline _posts/2015-03-23-CocoaDocs-Colo.md]: punctuation could use work here:

> In the [process](https://github.com/CocoaPods/cocoadocs.org/issues/320) of moving, I added one more feature: we now send data to the [CocoaPods status][7] page saying how many pods have been documented in the last 24hrs.

**Comment by jpsim** [comment]
Thanks for doing this, @orta!

**Comment by orta** [comment]
:D will make your changes on master

**Pull request merged**